lib/tropic/tropic_paragraph.flow

```diff
@@ -633,6 +633,11 @@ positionInteractiveWords(
 	genIndent : double,
 	joinedWordsConstructors : Tree<int, (string, [CharacterStyle]) -> Tropic>
 ) -> [Tropic] {
+	spaceCase = \word, viewFn, str, alBef, alAft -> {
```

Comment:
it might break tail recursion optimization. otherwise LGTM

Reply:
I doubt this optimization has significant effect anyway.